server/src/main/java/io/druid/server/coordinator/DruidCluster.java

```diff
@@ -70,6 +72,15 @@ public void add(ServerHolder serverHolder)
     return cluster.get(tier);
   }
 
+  public Set<ServerHolder> getAllServers()
+  {
+    final Set<ServerHolder> servers = new HashSet<>();
```

Comment:
I misunderstood that the same server can be involved in different tiers which it can't. I changed to list.